operator/Makefile

```diff
@@ -33,13 +33,14 @@ REGISTRY_BASE_OPENSHIFT = quay.io/openshift-logging
 REGISTRY_BASE ?= $(REGISTRY_BASE_COMMUNITY)
 
 # TODO(@periklis): Replace this image tag with VERSION once we have GH tags
-MAIN_IMAGE_TAG = main-39f2856
+MAIN_IMAGE_TAG = main-ee371a8
```

Comment:
Any news on when we will be getting tags?

Reply:
:shrug: not yet